uint8 motor_id
uint8 command
int64 data
---
uint8 result
string result_message
